src/locale/pages/assignees/en.yml

```diff
@@ -0,0 +1,9 @@
+queryStates:
+    error: This task must have a Smart Search query defined before it can be published.
```

Comment:
I think this could be phrased better. Something like:

> Your task will be targeted to people matching a Smart Search query. Before you can publish you must design the target group using Smart Search.

You can probably phrase it better than me. I just want it to be more user-focused and less technical.